src/Components/Dictionary/DictionaryCard.js

```diff
@@ -34,42 +34,41 @@ export default function DictionaryCard({
             : (
               <div>
                 {showMore ? def : `${def.substring(0, 250)}`}
-                <Button onClick={() => setShowMore(!showMore)}>
+                <Button style={{ textTransform: 'none', color: '#607AAD' }} onClick={() => setShowMore(!showMore)}>
```

Comment:
For future reference, it's best to move all of this kind of styling to the css file rather than having in-lines!